progressive-loader.js

```diff
@@ -29,25 +28,49 @@ module.exports = function loader (contentBuffer) {
     source = content.match(/^module.exports = (.*);/)[1]
   }
 
-  gm(path).size(function (err, size) {
-    if (err) {
-      console.error(err)
-      return
-    }
+  if (config.sharp) {
+    const sharpImg = require('sharp')(path)
+
+    sharpImg
+      .webp({ quality: 10 })
```

Comment:
should we change to specific format, or just use the same format?

Reply:
webp is only support by chrome, so it should definitely default to something else. You should run some examples through to see what ends up smaller, I don't recall the rest of the results from my original tests.

Reply:
I tested it. jpeg is smaller than tiff or png, also I have implemented it in https://apek.or.id/about